px-vis-behavior-common.html

```diff
@@ -1993,7 +1993,7 @@
       for(var i=0; i<this._uniqueIdsUsed.length; i++) {
         Px.uniqueIds.splice(Px.uniqueIds.indexOf(this._uniqueIdsUsed[i]), 1);
       }
-      this._uniqueIdsUsed = null;
```

Comment:
As you can see the suggestion here is to reset to the default literal value rather than null. I don't see this being an issue for memory leakage but I know there has to be a reason it was done this way to begin with.